addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/SourceCodeDisclosureGitScanRule.java

```diff
@@ -425,4 +426,31 @@ private boolean findSourceCodeGit(HttpMessage originalMessage) throws Exception
             return false;
         }
     }
+
+    private AlertBuilder createAlert(
+            byte[] disclosedData,
+            String filename,
+            String gitURIs,
+            HttpMessage originalMessage,
+            String uri) {
+        return newAlert()
+                .setConfidence(Alert.CONFIDENCE_MEDIUM)
+                .setUri(uri)
+                .setOtherInfo(new String(disclosedData))
+                .setEvidence(getEvidence(filename, gitURIs))
+                .setMessage(originalMessage)
+                .setRisk(getRisk());
+    }
+
+    @Override
+    public List<Alert> getExampleAlerts() {
+        return List.of(
+                createAlert(
+                                "disclosedData example".getBytes(),
+                                "exampleFile",
```

Comment:
If we are passing a value for the file name we should pass for the URI too (but like other info, evidence is not shown in docs though).

Reply:
Apparently i forgot to put it back in before making the commit. Will add it back (URI being empty seems to be the cause for the tests failling).

Reply:
I don't think that's the cause of the test failure, I will take a better look shortly to know what's missing.

Reply:
I'm not super sure either, but changing it made the tests work locally. I also made the other changes requested. If you find anything let me know, and thanks for the quick replies.

Reply:
Oh i see.